internal/cmd/gocdk/static/_assets/provision/blob/fileblob/fileblob.tf

```diff
@@ -0,0 +1,11 @@
+# TODO(rvangent): Add comments explaining.
```

Comment:
nit: These files are not `terraform fmt`-ed

Reply:
I did the best I could, but `terraform fmt` is not idempotent and gradually screws up comments :-(.